priv/repo/migrations/20220617170400_add_tire_pressures.exs

```diff
@@ -0,0 +1,35 @@
+defmodule TeslaMate.Repo.Migrations.AddTirePressures do
+  use Ecto.Migration
+
+  def change do
+    alter table(:positions) do
+      add_if_not_exists(:tpms_pressure_fl, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_fr, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_rl, :numeric, precision: 4, scale: 1)
+      add_if_not_exists(:tpms_pressure_rr, :numeric, precision: 4, scale: 1)
```

Comment:
Let's use the plain `add`s here. If a user's database already contains these columns, something is wrong.

```suggestion
      add(:tpms_pressure_fl, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_fr, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_rl, :numeric, precision: 4, scale: 1)
      add(:tpms_pressure_rr, :numeric, precision: 4, scale: 1)
```